weather/module.py

```diff
@@ -1,23 +1,155 @@
 from typing import Optional
+import aiohttp
 
+import nextcord
 from nextcord.ext import commands
 
 import pie.database.config
-from pie import check, i18n, logger
+from pie import check, i18n, logger, utils
+from pie.i18n.database import GuildLanguage, MemberLanguage
 
 from .database import Place
 
 _ = i18n.Translator("modules/fun").translate
 guild_log = logger.Guild.logger()
 config = pie.database.config.Config.get()
 
+# number of days to get forecast for (including current day, max is 3)
+NUM_OF_FORECAST_DAYS = 3
+# dict for getting the data from json easier (when you don't wan't some phase of day comment it)
+DAY_PHASES = {
+    "Morning": 2,
+    "Day": 4,
+    # "Evening": 6,
+    "Night": 7,
+}
+
 
 class Weather(commands.Cog):
     """Weather and forecast"""
 
-    def __init__(self, bot):
+    def __init__(self, bot: commands.Bot):
         self.bot = bot
 
+    def _get_useful_data(
+        self, json: dict, ctx: commands.Context, lang_preference: str
```

Comment:
I'm still not sure about the `json` variable name. `json` is usually reserved for the standard library, let's use `data` or something like that.